jui-core/index.d.ts

```diff
@@ -7,6 +7,10 @@
 
 declare var jui: JuiStatic;
 
+declare type ClassFunction = (() => any);
```

Comment:
i tried `{ new(): any }`

when i use '`new`' grammer, console show below message 

`
10:14  interface-over-type-literal  Use an interface instead of a type literal.
`

What should I fix?